### ISSUE TITLE
M> ONGOING: Quora Knowledge-Base Engagement  

### ISSUE THREAD

**Issue by ResonanceXX** [rchain/bounties#197]
Quora is a unique platform for knowledge gathering. The status of Rchain and Rhoc knowledge on the platform to the best of my knowing is non existent. 

An anonymous user asked a question. Working on sourcing concise answer. 

_Estimated Budget of Task:_ **$[500]**
_Estimated Timeline Required to Complete the Task:_ **[4 weeks]**
_How will we measure completion?_ **[Post Links and Screen shots]**

SMART

**Comment by ResonanceXX** [comment]
Other Coop members can add bits of information about the same thing.

**Comment by ResonanceXX** [comment]
Here the response

https://www.quora.com/What-is-rchain-all-about/answer/Christian-ThankGod?share=2bbef2f2&srid=50Tjl

**Comment by ResonanceXX** [comment]
Generating information demand? 

https://www.quora.com/anonymous/09650550f7604f448d5b2463aa0b1e3e

**Comment by kaka56** [comment]
The idea is so great because most questions are been answerable so people tend to  check the site in search 4 knowledge

**Comment by lapin7** [comment]
That’s true. Quora is great

**Comment by patrick727** [comment]
This is a great idea! I've been answering whatever blockchain questions on quora I could link to rchain and it gets consistent traffic see(and upvote):

[What is the best programming language to program the blockchain](https://www.quora.com/What-is-the-best-programming-language-to-learn-if-you-want-to-work-on-the-blockchain/answer/Patrick-Maguire-22)

[What is the next ethereum?](https://www.quora.com/What-is-the-next-Ethereum/answer/Patrick-Maguire-22)

[How does ethereum's Casper proof-of-stake work?](https://www.quora.com/How-does-Ethereums-Casper-proof-of-stake-protocol-work/answer/Patrick-Maguire-22)

**Comment by ResonanceXX** [comment]
Very impressive, must say. I'll get back with more. Cheers. 

Sent from Yahoo Mail on Android 
 
  On Thu, 7 Dec 2017 at 14:53, patrick727<notifications@github.com> wrote:   
This is a great idea! I've been answering whatever blockchain questions on quora I could link to rchain and it gets consistent traffic see(and upvote):

What is the best programming language to program the blockchain

What is the next ethereum?

How does ethereum's Casper proof-of-stake work?

**Comment by Mervyn853** [comment]
Great idea here @ResonanceXX , that will help to drive inbound traffic from a really targeted audience together with some decent inbound links. I will get started on some of these questions too.

**Comment by ResonanceXX** [comment]
Sure Merv. All the best while you're at it. 

Sent from Yahoo Mail on Android 
 
  On Thu, 7 Dec 2017 at 15:10, Mervyn853<notifications@github.com> wrote:   
Great idea here @ResonanceXX , that will help to drive inbound traffic from a really targeted audience together with some decent inbound links. I will get started on some of these questions too.

**Comment by kitblake** [comment]
Upvoted the answers on Quora. Please share any others that need some love.

**Comment by ResonanceXX** [comment]
@kitblakehttps://www.quora.com/anonymous/0ec36fefc26b461bba46ccde7d0a486e

Give this one a try

**Comment by kitblake** [comment]
Done. 
(Ha, I was worried that it was a techy question. I'm not technical. Responding to market questions is fine.)

**Comment by Mervyn853** [comment]
https://www.quora.com/What-is-an-Rchain-all-about/answer/Mervyn-Chng-2?srid=ulHUf

https://www.quora.com/What-is-an-RHOC-How-valuable-is-it-and-how-is-it-traded/answer/Mervyn-Chng-2?srid=ulHUf

https://www.quora.com/What-is-the-next-Ethereum/answer/Mervyn-Chng-2

Updated with the following answers, please help with upvoting :)

**Comment by Ojimadu** [comment]
Nice one, would upvote.

**Comment by ResonanceXX** [comment]
Have upvoted. Nicely done.

**Comment by ResonanceXX** [comment]
https://www.quora.com/unanswered/I-have-300-RHOCs-what-shoud-I-do-with-it


CAN WE GET ANSWERS TO THIS?

**Comment by ResonanceXX** [comment]
https://www.quora.com/anonymous/f141745174ae4add973acfe0d0f41574

**Comment by patrick727** [comment]
answered this one

https://www.quora.com/I-have-300-RHOCs-what-shoud-I-do-with-it/answer/Patrick-Maguire-22?prompt_topic_bio=1

**Comment by ResonanceXX** [comment]
@patrick727 saw that.

an an0nym0us user asked I answered t0 her
https://www.quora.com/I-have-300-RHOCs-what-shoud-I-do-with-it/comment/6499236

**Comment by ResonanceXX** [comment]
https://www.quora.com/unanswered/What-are-the-advantages-of-the-RHOC-cryptocurrency

an0ther

**Comment by lapin7** [comment]
@kaka56 Please send your email to ops@rchain.coop

**Comment by kaka56** [comment]
Thank you @lapin

**Comment by ResonanceXX** [comment]
If Bitcoin is cryptocurrency 1.0 and Ethereum is cryptocurrency 2.0, which is the cryptocurre… by Christian ThankGod https://www.quora.com/If-Bitcoin-is-cryptocurrency-1-0-and-Ethereum-is-cryptocurrency-2-0-which-is-the-cryptocurrency-3-0/answer/Christian-ThankGod?share=042c6654&srid=50Tjl

**Comment by ResonanceXX** [comment]
This answer has got RHOC mentioned on a similar thread where other up and coming cryptocurrency are mentioned on quora.
I expect a lot of exposure and inquiry.

**Comment by Ojimadu** [comment]
Please edit your answer. There was no ICO, RHOCs are not listed on coinbase and it's KuCoin not kukoin

**Comment by ResonanceXX** [comment]
Yeah about that. I based my answers more on what I felt. With the presiding authority to source more accurate answers. It is very important that people who have prior information assist in answering these questions. 

Sent from Yahoo Mail on Android 
 
  On Tue, 19 Dec 2017 at 18:17, Ojimadu<notifications@github.com> wrote:   
Please edit your answer. There was no ICO, RHOCs are not listed on coinbase and it's KuCoin not kukoin

**Comment by ResonanceXX** [comment]
Check to see if the revised answers are more conformant. 

Sent from Yahoo Mail on Android 
 
  On Tue, 19 Dec 2017 at 18:19, Kenechukwu Oliaku<kenechukwuoliaku@yahoo.com> wrote:   Yeah about that. I based my answers more on what I felt. With the presiding authority to source more accurate answers. It is very important that people who have prior information assist in answering these questions. 

Sent from Yahoo Mail on Android 
 
  On Tue, 19 Dec 2017 at 18:17, Ojimadu<notifications@github.com> wrote:   
Please edit your answer. There was no ICO, RHOCs are not listed on coinbase and it's KuCoin not kukoin

**Comment by ResonanceXX** [comment]
Thank you.

**Comment by Ojimadu** [comment]
I think you should remove the ICO thing and instead use "a private sale was held in October". Listing EtherDelta as an exchange too won't be bad even though some complain about it being a little technical.

**Comment by ResonanceXX** [comment]
@Ojimadu I have made these modifications. Listed etherdelta but made no mention of difficulty. I presume the other info is accurate and acceptable.

**Comment by Keaycee** [comment]
As collaborators (Team), it's part of our duty to be able to answer most of the questions on Quora. Quora  it's been known to impact knowledge on individuals world wide. I think participating or engaging in  Quora  questions about Rchain will enable people see the future in Rchain. I volunteer to be part of the Quora  collaborators.

**Comment by patrick727** [comment]
@Keaycee thanks, all you need to do is monitor quora and answer to the best of your abilities. 

I personally focus on the platform itself and the cooperative and less about the rhoc, because the rhoc is just a placeholder. see my comment on @ResonanceXX 's [link](https://www.quora.com/If-Bitcoin-is-cryptocurrency-1-0-and-Ethereum-is-cryptocurrency-2-0-which-is-the-cryptocurrency-3-0/answer/Christian-ThankGod?share=042c6654&srid=50Tjl)

**Comment by kitblake** [comment]
@ResonanceXX I looked at your Quora answer here
https://www.quora.com/If-Bitcoin-is-cryptocurrency-1-0-and-Ethereum-is-cryptocurrency-2-0-which-is-the-cryptocurrency-3-0/answer/Christian-ThankGod
and there is a bunch of stuff that doesn't make sense, or it's wrong. 
```
Cryptocurrency 3.0 has definitely, and with all utmost certainty got to be the “RHOC" This 
is coin based on the Block chain but with emphasis on a decentralized governance and 
interdependence system of transactions.
```
It's not about the RHOC token, it's about RChain. The second sentence doesn't make sense. Can you clarify it?
```
The RHOC has shown potential to rival and even surpass BTC in cryptocurrency 
desirability indexes. 
```
There is no such thing as a desirability index. Or can you link to one?
```
An independent sale will be achieved quite a while in the future, and another is planned 
by March next year upon the release of the Mercury platform.
```
This is misinformation. There is no sale planned in March and there is nothing definite planned upon the release of Mercury. 

Can you fix these things in your answer?

**Comment by ResonanceXX** [comment]
Have you only just seen this now. You should have made the correction a lot of time before. And above there's a point where similar calls were made. With corrections by me. You can supplant more information. This is marketing. That's by the way. 
Sent from Yahoo Mail on Android 
 
  On Fri, 12 Jan 2018 at 11:38, kitblake<notifications@github.com> wrote:   
@ResonanceXX I looked at your Quora answer here
https://www.quora.com/If-Bitcoin-is-cryptocurrency-1-0-and-Ethereum-is-cryptocurrency-2-0-which-is-the-cryptocurrency-3-0/answer/Christian-ThankGod
and there is a bunch of stuff that doesn't make sense, or it's wrong.
Cryptocurrency 3.0 has definitely, and with all utmost certainty got to be the “RHOC" This 
is coin based on the Block chain but with emphasis on a decentralized governance and 
interdependence system of transactions.

It's not about the RHOC token, it's about RChain. The second sentence doesn't make sense. Can you clarify it?
The RHOC has shown potential to rival and even surpass BTC in cryptocurrency 
desirability indexes. 

There is no such thing as a desirability index. Or can you link to one?
An independent sale will be achieved quite a while in the future, and another is planned 
by March next year upon the release of the Mercury platform.

This is misinformation. There is no sale planned in March and there is nothing definite planned upon the release of Mercury.

Can you fix these things in your answer?

**Comment by kitblake** [comment]
Yes, i only reviewed it now, and you should change it now. It's wrong.

**Comment by ResonanceXX** [comment]
You should consider making the call on quora. I'll leave it there till someone else makes an edit. Then I'll upvote that.  

Sent from Yahoo Mail on Android 
 
  On Fri, 12 Jan 2018 at 12:25, kitblake<notifications@github.com> wrote:   
Yes, i only reviewed it now, and you should change it now. It's wrong.

**Comment by Mervyn853** [comment]
@ResonanceXX I have highlighted those misleading information you have posted here https://www.quora.com/If-Bitcoin-is-cryptocurrency-1-0-and-Ethereum-is-cryptocurrency-2-0-which-is-the-cryptocurrency-3-0/answer/Christian-ThankGod and I had highlighted to you in private. Please change that as only have you the ability to.

**Comment by ResonanceXX** [comment]
I have made all corrections based on the recommendation above. There is no more misinformation. I Made sure to do a little encouragement for developers to join the coop on Github. Its marketing.

**Comment by kitblake** [comment]
Thanks @ResonanceXX. Yes, building community momentum is what we want to achieve.

**Comment by Ojimadu** [comment]
@ResonanceXX Here are my suggestions.
Replace this
```
Cryptocurrency 3.0 has definitely, and with all utmost certainty got to be the upcoming “REV" This is coin based on the Block chain but with emphasis on a decentralized governance and interdependence system of transactions.
```
With this
```
Cryptocurrency 3.0 has definitely, and with all utmost certainty has got to be RChain. RChain is a secure, scalable blockchain with emphasis on a decentralized governance and interdependence system of transactions.
```
This
```
Founded by Gregory Meredith, but with an ever growing population of contributors in Cooperative community in Github. The REV has shown potential to rival and even surpass BTC in cryptocurrency desirability indexes.
```
with this
```
Founded by Gregory Meredith, but with an ever growing population of contributors in Cooperative community in Github. The RChain has shown potential to rival and even surpass BTC in cryptocurrency desirability indexes.
```
This
```
Otherwise the RHOC is listed on KuCoin, EtherDelta and Can be traded seamlessly.
```
With this
```
RChain's issues 'RHOC' which is an ERC20 token that would be converted to "REV" on a 1:1 basis on laucnch of Mercury. RHOCs is listed on KuCoin, EtherDelta and can be traded seamlessly.
```
This
```
Making contributions to RChain on Github is a dream most developers ever chance upon.
```
With this
```
Anyone can make contributions to RChain on Github and get rewards in RHOCs by [signing up as a collaborator](https://github.com/rchain/Members/blob/master/CONTRIBUTING.md)

**Comment by Viraculous** [comment]
It has been awesome so far on members participation on this issue. I really think quora is great but better when internalized. What about a having an quora- type of platform within the co-op? 
Ref:  #243

**Comment by patrick727** [comment]
I commented on this answer a while ago

here is my response to the question, see if you get the tone and theme that we want to push for, more about the tech then anything else. 

https://www.quora.com/If-Bitcoin-is-cryptocurrency-1-0-and-Ethereum-is-cryptocurrency-2-0-which-is-the-cryptocurrency-3-0/answer/Patrick-Maguire-22?prompt_topic_bio=1

**Comment by ResonanceXX** [comment]
Me to earth.   @patrick727 


If Bitcoin is cryptocurrency 1.0 and Ethereum is cryptocurrency 2.0, which is the cryptocurre… by Christian ThankGod https://www.quora.com/If-Bitcoin-is-cryptocurrency-1-0-and-Ethereum-is-cryptocurrency-2-0-which-is-the-cryptocurrency-3-0/answer/Christian-ThankGod?share=042c6654&srid=50Tjl

**Comment by ResonanceXX** [comment]
Made a lot of effortless adjustments. Try and tone down the excess enthusiasm.

**Comment by ResonanceXX** [comment]
Beautiful answers. Welldone GG. I made sure to add supporting comments and describe personal experiences. 

https://www.quora.com/Will-RHOC-ever-climb-to-where-Bitcoin-is-now-Why-is-it-worth-it-to-invest-in-the-RHOC-at-all

Up-voted all answers too.

**Comment by ResonanceXX** [comment]
@patrick727 Is this a viable thread still? If its not in progress?

**Comment by ResonanceXX** [comment]
http://qr.ae/TU8EH7


With the bearish nature of BTC rates, a lot of inquires by coin holders for the next promising coin. I have explicitly explained that the RHOC is a placeholder for the REV, which is only functional upon Mercury release. Cheers.

**Comment by ResonanceXX** [comment]
An anonymous user asked 

https://www.quora.com/What-is-the-best-cryptocurrency-to-buy-as-a-person/answer/Christian-ThankGod



Was able to provide the answer below.

http://qr.ae/TU8jzG

**Comment by ResonanceXX** [comment]
So I encountered this question here.

https://www.quora.com/What-do-you-think-about-Bitcoin-blockchain-cryptocurrencies/answer/ResonanceXX

And my answer here.

http://qr.ae/TU8f4d

**Comment by ResonanceXX** [comment]
https://www.quora.com/Taking-in-account-the-fall-of-Bitcoins-price-what-are-the-alternative-options-of-Bitcoin-to-invest/answer/ResonanceXX#

**Comment by ResonanceXX** [comment]
cheers RChainers.

**Comment by ResonanceXX** [comment]
The slump in Bitcoin prices is generating a lot of consumer interest in Alt coins like ours. It is my genuine hope that is expressed in more stimulated exchange of the RHOC. 

And the emergence of RChain as a foremost player in the Blockchain space.

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1TsQ

**Comment by ResonanceXX** [comment]
https://www.quora.com/Why-is-the-Elastos-cryptocurrency-being-so-commented/answer/ResonanceXX

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1FWY

Just to share a monumental milestone in my engagement on Quora, the answers I delivered have been viewed about 1,800 times in 30 days. Although constancy has remained an issue (A lot of developmental challenges) but this has got to be pivotal and a reflection of growing interests on the RChain commodity out there. 
I also would wish for a Quora where the answers being requested from me are more about RChain, than are currently varied towards other Blockchain players too.

![screenshot-2018-3-30 stats - quora](https://user-images.githubusercontent.com/34219332/38120207-210b20b8-33bf-11e8-8c93-88ce6b2ee018.png)

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1oxw

The demand for Crypto related information is high, Especially with the bearish nature of BTC globally. 
With my esteemed position as a 'reliable information source' answering newbies question always come with an invitation to the Coop. Nationalitites have always varied, this due to the global reach of Quora.com
More updates should be expected.

**Comment by ResonanceXX** [comment]
I see a lot of quivering, but willingness to try out some other Blockchain ideas. Developers are domiciled in Quora, due to its informative nature. The reach has also multiplied, much to my surprise.

**Comment by ResonanceXX** [comment]
A user **Usher Blad** was curious about the future of Cryptocurrency in 5 years time. Without conceding any excitement, I interwove the positivity behind Mainnet launch in Q4 and the prospects of Our Chain going forward.

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1MxZ


Hello coop, responding to these questions have created a flurry of more requests. And there is plenty of fertile soil to plant the ''message'' of RChain cooperative.

**Comment by ResonanceXX** [comment]
New cryptocurrency enthusiasts are more readily attracted to the opportunity of learning more about a Blockchain enterprise, while earning a new form value (The RHOC).

http://qr.ae/TU1MD7
Here I delve in to subject answer required, and then subtly broach the topic of joining the coop.

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1MDo

Another answer for the inquisitive Brad Usla

**Comment by ResonanceXX** [comment]
Jaemin Linn had her question answered. And this has been an eventful answer with plenty of research. 
This has helped me fit into a niche with plenty of work to be done. 
I am grateful for the opportunity to do this and represent RChain

http://qr.ae/TU1MDO

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1MDa

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1929

An answer to Kim Boll's question.

**Comment by ResonanceXX** [comment]
http://qr.ae/TU192E

Investment opinion answers. in the #10,000 region.

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1JVs

Expert advice coyrtesy of the RChain cooperative member ResonanceXX. This is also a title that I boldly relay across to my readers all the while.

**Comment by ResonanceXX** [comment]
**Naib Shafi** has asked a question pertaining to value of coins and potential to attain a certain mark.
This is consistent with the many questions of members in the Telegram group for Price discussions. 

The potential of the coin, not the value of it is the main attraction of the inquirers. 

But I have always attempted to in a subtle but direct approach, to ensure that the potential of RChain's core work, which is birthing  a better Blockchain solution - The Mercury platform.

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1JVS

**Comment by ResonanceXX** [comment]
Yvonne Toledo a quora user, amidst the plentiful questions on demand today has asked **How can cryptocurrency make a success story?**

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1Jm9

A question requested by **Gordon ng** who subsequently upvotes the answer received. The genuine satisfaction is attained.

**Comment by ResonanceXX** [comment]
![screenshot-2018-3-30 stats - quora](https://user-images.githubusercontent.com/34219332/38774140-a8b96682-4058-11e8-937c-ec560c1096db.png)
Hurray!!

**Comment by ResonanceXX** [comment]
wait for it.....

**Comment by ResonanceXX** [comment]
![screenshot-2018-4-15 stats - quora](https://user-images.githubusercontent.com/34219332/38774142-c0b6d4ea-4058-11e8-9065-52116ca74916.png)
Voila RChain.
All thanks to you guys. I have known nothing more than I am.

**Comment by ResonanceXX** [comment]
There is plenty of activity on the Alt coin discussions board. And Mining is a big subject. But RChain always fits into the replies. I am also glad to contribute via Pithia, regarding inquiry in to futuristic Blockchain companies.

**Comment by ResonanceXX** [comment]
And so I put the link to the live stream on Quora. I suppose that any interested person can be allowed to watch, and not just cooperative members.

**Comment by ResonanceXX** [comment]
There is presently a snacks break.

**Comment by ResonanceXX** [comment]
**Steven Che** requested an answer on the volatility of Bitcoin.

**Comment by ResonanceXX** [comment]
Working to supply a positive answer to that.

**Comment by ResonanceXX** [comment]
http://qr.ae/TU16Gp

Check the comments section to see a subtle invitation to the coop.

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1ENQ

**Comment by ResonanceXX** [comment]
The demand for Altcoins is at an ATH. RHOC- steadying the information pipeline.

**Comment by ResonanceXX** [comment]
Trying to shore up the significance of the rhoc as a placeholder for the rev. And that Mercury release is an important date to keep.
http://qr.ae/TU1jEi

**Comment by ResonanceXX** [comment]
http://qr.ae/TU1dMO

**Comment by ResonanceXX** [comment]
Hello, in the response to the question, the usefulness of Resonate (our music streaming solution) seemed a perfect example for the multifarious use of the Blockchain.

http://qr.ae/TU1d5u

I also made an example of lifeID's solution on identity and security to illustrate the varied aspects of blockchain tech in improving many areas of life.

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTXfW

**Comment by ResonanceXX** [comment]
![screenshot-2018-4-28 stats - quora](https://user-images.githubusercontent.com/34219332/39397931-d17babec-4afe-11e8-9677-124fd1672602.png)

**Comment by ResonanceXX** [comment]
Honestly, working this hard, in a new solution has been exciting.

**Comment by ResonanceXX** [comment]
In the past one week. The numbers so far.
![screenshot-2018-5-5 stats - quora](https://user-images.githubusercontent.com/34219332/39665460-a36b3ad6-508c-11e8-8bd9-c2cff4612df4.png)

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTt0V

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTnJx

Sowing little seeds of information.

**Comment by ResonanceXX** [comment]
![screenshot-2018-5-7 michael thayer upvoted your answer to when you buy btc on coinbase or other markets how do they regist](https://user-images.githubusercontent.com/34219332/39704287-beca0a6a-5202-11e8-91ee-40cd7825a0ad.png)

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTyek

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTCcU

**Comment by ResonanceXX** [comment]
https://medium.com/rchain-cooperative/rnode-v0-3-1-introduces-the-rchain-consensus-protocol-23e67666ba82

This is good news. I am including the link only and giving full credit to @Nash Foster. The milestone effect will validate my precious assertions that Mercury launch is impending.

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTF9H

My longest answer in a while. Users like him can be given better information. Rchain may be their first inroads into the crypto space.

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTFiY

**Comment by ResonanceXX** [comment]
![screenshot-2018-5-15 write answers - quora](https://user-images.githubusercontent.com/34219332/40055241-2805edae-583f-11e8-9aed-0af3c0bd5faa.png)

**Comment by ResonanceXX** [comment]
Plenty demand, lots of hindrances for now. Looking to start a more specific approach. Zero-ing in on the developer void that RChain may need to fill.

**Comment by ResonanceXX** [comment]
**Mateus Korcioski** has requested an answer to - What factors prevent cryptocurrency's mainstream adoption/usage?

P.S - Quite frankly for me, and in accordance to the solution that we are about as a project. Scalability has got to be one major reason for this.
Looking to, twirl in a better answer and quip in couple of links to drive the point home.

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTklc

**Comment by ResonanceXX** [comment]
**My answer to patrick jorgensen's question about coin nomenclature**

http://qr.ae/TUTumw

**Comment by ResonanceXX** [comment]
**Kumail Naqvi** requested I share some info on the state of the ecosystem presently.

http://qr.ae/TUTuqH Looking to be an optimist but spiked up a bit of realism too.

**Comment by pmoorman** [comment]
@ResonanceXX same here as #607 : if we don't see good ROI from Quora, I suggest we close this off as a failed experiment, and move on to greener pastures.

What's your take on Quora effectiveness? My vote would be to close this off, **unless we can demonstrate the ROI.**

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTgpV

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTxnD

Adopting @lapin7's original link list system.

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTxrU

**Jessica few** Question requested and answered. All the links being there.

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTxy9

New method. I expect newer and more better results.

**Comment by ResonanceXX** [comment]
New approach. Reaching out to the ecosystem of core Blockchain enthusiasts. 
Being around a topic as **How can I start a career in blockchain technology?**

Then responding suitably. With all the links there.

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTxU9

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTmTK

**Comment by ResonanceXX** [comment]
One of the key goals of any marketing strategy is to ensure as much market penetration as possible for the product/service.

**Comment by Pionelle** [comment]
This is good.I followed the link and it shows you already have 3.5k answer views this month and 13k views in total.
You've also been consistent with what you put up!Well done! 

    On Monday, May 28, 2018 5:14 AM, Chris Ken <notifications@github.com> wrote:
 

 One of the key goals of any marketing strategy is to ensure as much market penetration as possible for the product/service.—

**Comment by ResonanceXX** [comment]
@Pionelle Thank you very much. You can contribute in your own ways too. Cheers.

**Comment by ResonanceXX** [comment]
@All By doing this, many people get to know about our company, what we offer and how to interact with these solutions and services.

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTlBb

**Comment by ResonanceXX** [comment]
![screenshot-2018-5-26 chris ken s answer to what makes ripple different than other cryptocurrencies - quora](https://user-images.githubusercontent.com/34219332/40632145-a6b04a9a-62db-11e8-9f39-061f0c5a5a60.png)
Plenty of activity around Quora. Although primary interest not RChains'. Responses from @Chris.Ken imply RChain and end with supporting links.

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTlEN

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTzTA

**Comment by ResonanceXX** [comment]
http://qr.ae/TUT6qy

**Comment by ResonanceXX** [comment]
http://qr.ae/TUT66M

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTBGU

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTBGE

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTjoo

**Comment by ResonanceXX** [comment]
Stats for the week.

**Comment by ResonanceXX** [comment]
![screenshot-2018-6-1 stats - quora](https://user-images.githubusercontent.com/34219332/40868877-d825d4a6-6609-11e8-858a-614b4ba36815.png)

**Comment by ResonanceXX** [comment]
http://qr.ae/TUTLJA

**Comment by ResonanceXX** [comment]
Another shift gone by.

**Comment by ResonanceXX** [comment]
http://qr.ae/TUp7t5

**Comment by ResonanceXX** [comment]
![screenshot-2018-6-6 stats - quora](https://user-images.githubusercontent.com/34219332/41009218-83168caa-6927-11e8-835c-bb34cc52146d.png)

**Comment by ResonanceXX** [comment]
![screenshot-2018-6-6 home - quora](https://user-images.githubusercontent.com/34219332/41042809-7e3a13d2-699a-11e8-8239-b54e7320c06f.png)

**Comment by ResonanceXX** [comment]
[This is an invite/link dropping](http://qr.ae/TUppda)

**Comment by ResonanceXX** [comment]
Jigar Modi is an app marketing strategist with a wonderful list on Medium, published by Hacker Noon - [Top 10 Blockchain companies](https://hackernoon.com/top-10-blockchain-technology-companies-cc1666c9da7e) Talks with him to have RChain listed with a small writeup and supporting links have been initiated.

**Comment by ResonanceXX** [comment]
And also cold texting - or so - some Blockchain developer specific threads on Quora, spreading the word to a more specific audience.

**Comment by ResonanceXX** [comment]
http://qr.ae/TUpvy7

**Comment by ResonanceXX** [comment]
http://qr.ae/TUpvA9

**Comment by ResonanceXX** [comment]
http://qr.ae/TUpK09

**Comment by ResonanceXX** [comment]
![screenshot-2018-6-13 home - quora](https://user-images.githubusercontent.com/34219332/41369904-1dd4ec84-6f3e-11e8-90a7-e4f2734b96c9.png)

**Comment by ResonanceXX** [comment]
http://qr.ae/TUpsMh

**Comment by ResonanceXX** [comment]
http://qr.ae/TUpsMJ

**Comment by ResonanceXX** [comment]
http://qr.ae/TUps58

**Comment by ResonanceXX** [comment]
http://qr.ae/TUps5h

**Comment by ResonanceXX** [comment]
![screenshot-2018-6-20 stats - quora](https://user-images.githubusercontent.com/34219332/41655107-b80643d0-7483-11e8-95e7-5f11fd1a9f1c.png)

**Comment by ResonanceXX** [comment]
http://qr.ae/TUpudA

**Comment by ResonanceXX** [comment]
http://qr.ae/TUpw5p

**Comment by ResonanceXX** [comment]
http://qr.ae/TUpUCo

**Comment by AyAyRon-P** [comment]
@ResonanceXX changed the title to ONGOING

**Comment by ResonanceXX** [comment]
http://qr.ae/TUpUCo
![screenshot-2018-6-29 home - quora](https://user-images.githubusercontent.com/34219332/42089154-ff3d1da2-7b93-11e8-9020-ba7e63773d94.png)

**Comment by pmoorman** [comment]
@ResonanceXX I've been looking through the Squarespace analytics on rchain.coop (the main website), but in the traffic sources I can't see any significant number from Quora traffic (which means it's below 5 referral hits from Quora in the last 30 days).

I'd suggest that this strategy is not reaching it's intended purpose & ROI. Looking at the strategy employed, I think the reason is obvious: there is little connect between the question and a reason to go visit the RChain website.

I'll vote $100 for this month for the effort, and suggest we stop this strategy because it's proven to not really generate results.

cc @kitblake @AyAyRon-P

**Comment by ResonanceXX** [comment]
@pmoorman Hello there? 

Sent from Yahoo Mail on Android 
 
  On Fri, 29 Jun 2018 at 16:05, pmoorman<notifications@github.com> wrote:   
Closed #197.
